routes/index.js

```diff
@@ -17,18 +17,23 @@ var mongoose = require('mongoose');
 // Makes connection asynchronously.  Mongoose will queue up database
 // operations and release them when the connection is complete.
 mongoose.connect(config.mongodbURI, { useMongoClient: true }, function (err, res) {
-  if (err) {
-    console.log ('ERROR connecting to: ' + config.mongodbURI + '. ' + err);
-  } else {
-    console.log ('Succeeded connected to: ' + config.mongodbURI);
-  }
+    if (err) {
+        console.log('ERROR connecting to: ' + config.mongodbURI + '. ' + err);
+    } else {
+        console.log('Succeeded connected to: ' + config.mongodbURI);
+    }
 });
 
-require('../models/AssignmentCallback');
-require('../models/WorkspaceEvent');
```

Comment:
I think these lines are incorrect, I just added them as part of #16. The rest of it seems to be there though.